Project.toml

```diff
@@ -5,14 +5,22 @@ version = "0.1.0"
 
 [deps]
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
+JuliaFormatter = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+OhMyREPL = "5fb14364-9ced-5910-84b2-373655c76a03"
 RowEchelon = "af85af4c-bcd5-5d23-b03a-a909639aa875"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+TerminalPager = "0c614874-6106-40ed-a7c2-2f1cd0bff883"
```

Comment:
@HettieC these are probably deps of your environment, but not of the package.

Maybe you want to make a different environment directory to test the stuff. Or do not commit the changes in Project.toml.

Reply:
(I fixed that in `main`)